### DISCUSSION TITLE
Can't change fonts for titles and headers?

### DISCUSSION THREAD

**Discussion by Taliesin1** [SlRvb/Obsidian--ITS-Theme#220] [Q&A]
Is it possible to use custom fonts for titles and headers? I'm trying to enter my own fonts that I have loaded in my system, but they won't take. Trying, for example, in Workspace > Vault Title > Vault Font and Note > Header Font. I can only get the default fonts to work.

**Comment by SlRvb** [comment]
They should work as long as the font names installed are the same names you're writing in those style settings fields 🤔 Could you screenshot the style settings you're writing and the font's name you're trying to use?

**Comment by Taliesin1** [comment]
Thanks! Here’s a screenshot showing everything from all angles — the Style Settings plug-in setting for the Heading font, my font manager showing that the font is active, and the name of the font, "OldNewspaperTypes," along with a preview of what the font should look like, and a note popped out into its own window so you can see that the font is not being applied. I think that's illustrates everything. Thanks so much for looking at this. Probably user error on my part!

<img width="2044" alt="Screenshot 2023-07-03 at 9 42 58 PM" src="https://github.com/SlRvb/Obsidian--ITS-Theme/assets/10515719/6bfa4526-78b2-4528-b8a1-d4f911a2f128">

**Reply by Taliesin1** [reply]
UPDATE: I see that the font is actually working — just not where I expected it. The "Shadowdark" vault name in the top left is taking the font (just noticed that). However, it's not appearing where I expect, which is in the H1. ("Pegasus" is my screenshot). How do I get the font to take there as well? I thought since I was applying it to "Headings"....
